client/lib/plans/constants.js

```diff
@@ -173,6 +173,16 @@ const WPComGetBillingTimeframe = abtest => {
 	return i18n.translate( 'per month, billed yearly' );
 };
 
+const WPComGetBiennialBillingTimeframe = abtest => {
+	if ( abtest ) {
+		if ( isEnabled( 'upgrades/2-year-plans' ) && abtest( 'multiyearSubscriptions' ) === 'show' ) {
+			return i18n.translate( '/month, billed biennially' );
```

Comment:
Hi! I've found a possible matching string that has already been translated **17** times:
`translate( '/month, billed annually or biennially' )` *ES Score: 22*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).